### ISSUE TITLE
Feature request: "ReDoc initialized!" callback?

### ISSUE THREAD

**Issue by hlubovac** [Rebilly/ReDoc#381]
I hope it's clear. Looking to execute some custom script once the content is ready.

Thanks!
Hari

**Comment by unional** [comment]
Looking for this feature also. Mainly for testing.

Also, currently the `ReDoc Initialized!` is sent to `console.log()` before any parsing error is reported.
It would be nice to have a way to determine if the spec is loaded and parsed correctly (thus the spec is valid).

**Comment by RomanGotsiy** [comment]
@unional this will be available in the upcoming v2 release.
Also, I will try to fix console.log timing in the 1.x version to be more useful.
